crates/biome_css_analyze/tests/specs/nursery/noInvalidPositionAtImportRule/invalid.css.snap

```diff
@@ -0,0 +1,28 @@
+---
+source: crates/biome_css_analyze/tests/spec_tests.rs
+expression: invalid.css
+---
+# Input
+```css
+a {}
+@import 'foo.css';
+
+```
+
+# Diagnostics
+```
+invalid.css:2:2 lint/nursery/noInvalidPositionAtImportRule ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Unexpected import position.
+  
+    1 │ a {}
+  > 2 │ @import 'foo.css';
+      │  ^^^^^^^^^^^^^^^^^
```

Comment:
Not sure if you noticed, but the diagnostic isn't correct because it doesn't highlight the `@`.

Reply:
Thanks for the review.
Sorry, I didn't know how to do it 🙇‍♂️
Is there a sample somewhere?